packages/cadl-rlc-test/cadl-output/package.json

```diff
@@ -0,0 +1,84 @@
+{
+  "name": "@msinternal/pet-store",
+  "sdk-type": "client",
+  "author": "Microsoft Corporation",
+  "version": "1.0.0-beta.1",
+  "description": "Pet Store Service",
+  "keywords": ["node", "azure", "cloud", "typescript", "browser", "isomorphic"],
+  "license": "MIT",
+  "main": "dist/index.js",
+  "module": "./dist-esm/index.js",
+  "types": "./types/pet-store.d.ts",
+  "repository": "github:Azure/azure-sdk-for-js",
+  "bugs": { "url": "https://github.com/Azure/azure-sdk-for-js/issues" },
+  "files": [
+    "dist/",
+    "dist-esm/",
+    "types/pet-store.d.ts",
+    "README.md",
+    "LICENSE",
+    "review/*"
+  ],
+  "//metadata": {
```

Comment:
@qiaozha Could you suggest we need this metadata in package.json from CADL?

Reply:
This is need by the sdk repo. I think you can remove it if azureSdkForJs is false.